### PR TITLE
Only show "did you know?" popups once per session.

### PR DESCRIPTION
**Pull request by bhollis**
Plus, offer a settings button to bring back all previously hidden tips. I also moved the CSV export buttons down to the bottom of the settings page while I was at it.

### CODE REVIEW

**Comment by bhollis** [comment]
This fixes #818.

File: app/scripts/services/dimInfoService.factory.js [hunk 1]
```diff
@@ -53,6 +53,12 @@
           }
           content.func();
         });
+      },
+      // Remove prefs for "don't show this again"
+      resetHiddenInfos: function() {
+        SyncService.get().then(function(data) {
+          SyncService.set(_.omit(data, (v, k) => k.startsWith('info.')), true);
```
Comment (kyleshay) [inline app/scripts/services/dimInfoService.factory.js]: nice!! totally forgot we prefixed those! :D

**Comment by kyleshay** [comment]
super cool!! definitely was on the to-do list! :D thank you ben :)

File: app/scripts/move-popup/dimMovePopup.directive.js [hunk 2]
```diff
@@ -120,16 +120,21 @@
       });
     };
 
-    /**
-     * Move the item to the specified store. Equip it if equip is true.
-     */
-    vm.moveItemTo = dimActionQueue.wrap(function moveItemTo(store, equip) {
+    // Only show this once per session
+    const didYouKnow = _.once(() => {
```
Comment (kyleshay) [inline app/scripts/move-popup/dimMovePopup.directive.js]: TIL: http://underscorejs.org/#once

love it!

**Comment by SunburnedGoose** [comment]
Look good.

**Pull request merged**